src/main/java/org/jboss/windup/plugin/WindupMojo.java

```diff
@@ -139,76 +161,50 @@ public void execute() throws MojoExecutionException
                 e.printStackTrace();
             }
         }
+
         if (userRulesDirectory != null)
             windupConfiguration.addDefaultUserRulesDirectory(Paths.get(userRulesDirectory));
 
-
         if (userIgnorePath == null)
-        {
-            // RM: Modify by upgrade to WindUp latest version
-            // userIgnorePath = WindupPathUtil.getWindupIgnoreListDir().toString();
-            userIgnorePath = PathUtil.getWindupIgnoreDir().toString();
-        }
+        	userIgnorePath = PathUtil.getWindupIgnoreDir().toString();
+        windupConfiguration.addDefaultUserIgnorePath(Paths.get(userIgnorePath));
 
-        if (userIgnorePath != null && !Files.isDirectory(Paths.get(userIgnorePath)))
-        {
-            try
-            {
-                Files.createDirectories(Paths.get(userIgnorePath));
-            }
-            catch (IOException e)
-            {
-                e.printStackTrace();
-            }
-        }
-
-        if (userIgnorePath != null)
-            windupConfiguration.addDefaultUserIgnorePath(Paths.get(userIgnorePath));
-
-        FileUtils.deleteQuietly(windupConfiguration.getOutputDirectory().toFile());
-        Path graphPath = windupConfiguration.getOutputDirectory().resolve("graph");
-
-        Furnace furnace = getFurnace();
-        try
-        {
-            start(true, true, furnace);
 
-            install("org.jboss.forge.addon:addon-manager," + forgeVersion, true, furnace);
-            install("org.jboss.forge.addon:maven," + forgeVersion, true, furnace);
-            install("org.jboss.forge.addon:projects," + forgeVersion, true, furnace);
-            install("org.jboss.windup.ui:windup-ui," + windupVersion, true, furnace);
-            install("org.jboss.windup.rules.apps:windup-rules-java," + windupVersion, true, furnace);
-            install("org.jboss.windup.rules.apps:windup-rules-java-project," + windupVersion, true, furnace);
-            install("org.jboss.windup.rules.apps:windup-rules-java-ee," + windupVersion, true, furnace);
-            install("org.jboss.windup:windup-tooling," + windupVersion, true, furnace);
-            install("org.jboss.windup.rules.apps:windup-rules-tattletale," + windupVersion, true, furnace);
+        Furnace furnace = createAndStartFurnace();
 
-            AddonRegistry addonRegistry = furnace.getAddonRegistry();
-            WindupProcessor windupProcessor = addonRegistry.getServices(WindupProcessor.class).get();
+        install("org.jboss.forge.addon:core,"+forgeVersion, true, furnace);
+        //install("org.jboss.forge.addon:furnace,"+forgeVersion, true, furnace);
```

Comment:
Right, this PR is work-in-progress / for discussion. (There are no labels here).